README.adoc

```diff
@@ -286,7 +286,7 @@ query/server.xml
 include::finish/query/src/main/liberty/config/server.xml[]
 ----
 
-Add the [hotspot=grpc file=3]`grpc` and [hotspot=grpcClient file=3]`grpcClient` features to the Liberty `server.xml` configuration file for the `query` service. These two features enable gRPC server and client support on Liberty respectively. Configure the [hotspot=grpcClientConfig file=3]`grpcClient` feature with the `headersToPropagate` and `target` attributes. If you want to learn more about `grpcClient` feature configuration, see the https://openliberty.io/docs/latest/reference/config/grpcClient.html[GRPC Client Properties^].
+Add the [hotspot=grpcClient file=3]`grpcClient` feature to the Liberty `server.xml` configuration file for the `query` service. This feature enables gRPC client support on Liberty. Configure the [hotspot=grpcClientConfig file=3]`grpcClient` element with the `maxInboundMessageSize` attribute to restrict inbound messages to 1024 bytes. This configuration applies universally to all gRPC client calls that get made, as indicated by the wildcard (`*`) in the `host` attribute. If you want to learn more about `grpcClient` element configuration, see the https://openliberty.io/docs/latest/reference/config/grpcClient.html[GRPC Client Properties^].
```

Comment:
"applies universally to all gRPC client calls that get made" ... client calls are not inbound?